src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -83,10 +102,10 @@
       + "initial connection to discover the full cluster membership (which may change "
       + "dynamically), "
       + "this list need not contain the full set of servers (you may want more than one, though, "
-      + "in case a server is down).";
+      + "in case a server is down). While it is required to configure either of this or "
+      + "``zookeeper.connect``; it is recommended to always use ``bootstrap.servers``.";
 
-  public static final String BOOTSTRAP_SERVERS_DEFAULT = "PLAINTEXT://localhost:9092";
-  public static final String KAFKACLIENT_CONNECTION_URL_DEFAULT = "localhost:2181";
+  public static final String BOOTSTRAP_SERVERS_DEFAULT = "";
```

Comment:
there is a blank line separating BOOTSTRAP_SERVERS_DEFAULT from the other BOOTSTRA_SERVERS constants, but no new line separating it from SCHEMA_REGISTRY_URL constants. reverse this ...